resources/test/README.md

```diff
@@ -40,8 +49,26 @@ This should be followed by one or more `<script>` tags that interface with the
       }, 'This test is expected to fail.');
     </script>
 
-Finally, each test may include a summary of the expected results as a JSON
-string within a `<script>` tag with an `id` of `"expected"`, e.g.:
+### Unit tests
+
+The "unit test" type allows for concisely testing the expected behavior of
+assertion methods. These tests may define any number of sub-tests; the
+acceptance criteria is simply that any tests executed pass.
+
+### Functional tests
+
+In order to test the behavior of the harness itself, some tests must force
+Thoroughly testing the behavior of the harness itself requires ensuring a
```

Comment:
I think you forgot to remove a line here.

Reply:
You're right--thanks for catching that!